### ISSUE TITLE
loss is negative

### ISSUE THREAD

**Issue by xueyushan** [HKUST-Aerial-Robotics/Stereo-RCNN#21]
Why is the loss negative? 
What do the plus or minus of the loss represent?
Thanks very much!

**Comment by xueyushan** [comment]
@Peiliang

**Comment by PeiliangLi** [comment]
It's normal. Because each loss is weighted by their uncertainty: loss = sum(loss_i * torch.exp(-uncert[i]) + uncert[i]). As the loss_i becomes smaller, the uncertainty will become negative.

**Comment by xueyushan** [comment]
The larger the absolute value of the loss,the network is in convergence?

**Comment by PeiliangLi** [comment]
The optimizing variable is the loss instead of abs(loss).
